src/app/debug/duck/selectors.ts

```diff
@@ -146,15 +146,21 @@ const getResourceStatus = (debugRef: IDebugRefRes): IDerivedDebugStatusObject =>
       };
     }
     case 'DirectVolumeMigrationProgress': {
-      const { conditions } = debugRef.value.data.object.status;
+      const {
+        errors,
+        warnings,
+        phase,
+        conditions,
+        totalProgressPercentage,
+      } = debugRef.value.data.object.status;
       const hasWarning = conditions?.some((c) =>
         checkListContainsString(c.category, warningConditionTypes)
       );
       const hasFailure = conditions?.some((c) =>
         checkListContainsString(c.type, ['InvalidPod', 'InvalidPodRef'])
       );
-      const hasCompleted = false;
-      const hasRunning = false;
+      const hasCompleted = phase === 'Succeeded';
+      const hasRunning = totalProgressPercentage !== '100%';
```

Comment:
@ibolton336 it's been my experience that there are cases where totalProgressPercentage may not reach 100% if we don't poll the Rsync Pod logs one last time before it shuts down with Phase=Succeeded. @pranavgaikwad does this sound accurate to you?

I just want to make sure we show "hasCompleted" with greater precedence than "hasRunning".